packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md

```diff
@@ -0,0 +1,178 @@
+
+    See the parameter [Details](/docs/reference/api/apps/#details) section on the Apps API Reference page for configuration guidance on the use of wildcard subdomains.
+
+1. Add the **Base URI** of your application during local development, such as `http://localhost:3000`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
+1. Assign the group that you want (if you set Group Assignments for your app) or leave the **Everyone** default. See the [Assign app integrations](https://help.okta.com/okta_help.htm?id=ext_Apps_Apps_Page-assign) topic in the Okta product documentation for instructions on how to assign the app integration to individual users and groups.
+1. Click **Save** to finish creating the Okta app integration.
+1. On the **General** tab, the **Client Credentials** section shows the client ID and client secret values for your app integration.
+1. Copy the **Client ID** and **Client secret** values using the **Copy to Clipboard** button beside each text field.
+You need to copy some values into your application later, so leave your Admin Console open.
+1. In the **General Settings** section, click **Edit** and scroll down to **Login**. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow. When Okta redirects to this endpoint (for example, `https://example.com/login`), the client is triggered to send an authorize request. This URI is also used when users reset their passwords while signing in to the app. Okta redirects the user back to this URI after the password is reset so that the user can continue to sign in.
+
+### Enable refresh token rotation
+
+You can choose to [get a refresh token](/docs/guides/refresh-tokens/get-refresh-token/) along with the access token and/or ID token.
+
+The default refresh token behavior is **Use persistent token** for web apps.
+
+Rotating refresh tokens is an <ApiLifecycle access="ea"/> feature.
+You can enable an [Early Access (Self-Service)](/docs/reference/releases-at-okta/#early-access-ea) feature for your org in the **Settings** > **Features** page inside the Admin Console.
+
+To enable refresh token rotation in your app integration, do the following:
+
+1. Open the web app integration that you just created and select the **General** tab.
+1. Scroll to the **General Settings** panel, and click **Edit**.
+1. In the **Allowed grant types**, select **Refresh Token**.
+1. In the **Refresh Token** section, select **Rotate token after every use**.
+    > **Note:** The default number of seconds for the **Grace period for token rotation** is set to 30 seconds. You can [change the value](/docs/guides/refresh-tokens/refresh-token-rotation/#enable-refresh-token-rotation) to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token. Using a value of 0 indicates that there is no grace period.
```

Comment:
When built, there's no space between step and the note.

Reply:
I've added a space, and also deindented it so that it appears at the same indent level as the note further below.

Reply:
added